### PR TITLE
Change facebook top tile to github and improve styling of top tiles icons

### LINKED ISSUE #14072: Replace Facebook top site tile with GitHub and improve top site tiles styling
### Description
The Facebook logo should be changed to something different, I myself would recommend github sice a lot of brave users might want to use GitHub.

### Additional Information
cc @bradleyrichter

### PR DESCRIPTION
**Pull request by Jacalz**
Fixes #14072 

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

### CODE REVIEW

File: js/data/newTabData.js [hunk 1]
```diff
@@ -26,19 +26,19 @@ module.exports.topSites = [
...
+    'key': 'https://github.com/brave/|0',
     'count': 0,
-    'favicon': `${iconPath}/facebook.png`,
-    'location': 'https://www.facebook.com/BraveSoftware/',
-    'themeColor': 'rgb(59, 89, 152)',
-    'title': 'Brave Software | Facebook'
+    'favicon': `${iconPath}/github.png`,
+    'location': 'https://github.com/brave/',
+    'themeColor': 'rgb(255, 255, 255)',
+    'title': 'Brave Software | Github'
```
Comment (bsclifton) [inline js/data/newTabData.js]: this should be `GitHub`

Comment (Jacalz) [reply js/data/newTabData.js]: Oh, my bad 😅 Will fix 🙂

Comment (Jacalz) [reply js/data/newTabData.js]: This is fixed now :+1:

File: js/data/newTabData.js [hunk 2]
```diff
@@ -63,6 +63,6 @@ module.exports.topSites = [
     'favicon': `${iconPath}/playstore.png`,
     'location': 'https://play.google.com/store/apps/details?id=com.brave.browser',
     'themeColor': 'rgb(241, 241, 241)',
-    'title': 'Brave Browser: Fast AdBlock – Apps para Android no Google Play'
+    'title': 'Brave Browser: Fast AdBlock – Apps for Android not Google Play'
```
Comment (NejcZdovc) [inline js/data/newTabData.js]: If I go to the link above it says `Brave Browser: Fast AdBlocker - Apps on Google Play`

Comment (NejcZdovc) [reply js/data/newTabData.js]: @Jacalz

Comment (Jacalz) [reply js/data/newTabData.js]: Fixed now with latest commit :+1:

**Comment by Jacalz** [comment]
I noticed that the playstore logo was outdated, so I updated it also :+1: 
Maby you could re-review this? @bradleyrichter

**Comment by Jacalz** [comment]
This is the state of this PR at the moment:
image

**Comment by Jacalz** [comment]
Thanks for merging 🙏

**Pull request merged**